src/settings_import_export.py

```diff
@@ -58,11 +58,11 @@ def export_settings(window, path):
         with open(path, 'w') as file:
             for line in gsettings_list.split('\n'):
                 file.write(' '.join(line.split(' ')[1::]) + '\n')
-        toast_msg = 'File successfully saved'
+        toast_msg = _('File successfully saved')
 
     except Exception as e:
         print('Can\'t export settings to file: ' + path)
         print(e)
-        toast_msg = 'Failed to save file'
+        toast_msg = _('Failed to save file')
 
-    Adw.PreferencesWindow.add_toast(window, Adw.Toast.new(toast_msg))
+    window.add_toast(window, Adw.Toast.new(toast_msg))
```

Comment:
do not pass window as argument
`window.add_toast(Adw.Toast.new(toast_msg))`

Reply:
when you used `Adw.PreferencesWindow.add_toast(window, Adw.Toast.new(toast_msg))`, you called a class method that required an instance as the first argument, but when the method is called on the instance itself, you don't need to pass it as an argument: `window.add_toast(window, Adw.Toast.new(toast_msg))` = `Adw.PreferencesWindow.add_toast(window, window, Adw.Toast.new(toast_msg))`, this causes an error:
```
TypeError: Adw.PreferencesWindow.add_toast() takes exactly 2 arguments (3 given)
```

Reply:
That makes sense. I'll correct it right away.